src/lib/components/Avatar/index.tsx

```diff
@@ -28,10 +28,11 @@ const AvatarComponent: FC<AvatarProps> = ({
   statusPosition = 'top-left',
   ...props
 }) => {
+  const theirProps = Object.entries(props).filter(([key]) => key !== 'className');
```

Comment:
Ah-ha... Great! Thanks.
Don't you think that we are going to reuse this in other components as well? If so, isn't better to create a helper?

Reply:
Yup!